frontend/src/components/ItemPage.tsx

```diff
@@ -21,6 +21,7 @@ function ItemPage() {
     const fetchData = () => {
         axios.get("/api/items/")
             .then(response => {
+                console.log(response.data)
```

Comment:
you dont need console.log in prod code :S

Reply:
thank u :D haha i forgot that <3